src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java

```diff
@@ -249,35 +250,35 @@ public class ProGuardMojo extends AbstractMojo {
 	private boolean attachSeed;
 
 	/**
-	 * Specifies attach artifact type
+	 * Specifies attach artifact type.
 	 *
 	 * @parameter default-value="jar"
 	 */
 	private String attachArtifactType;
 
 	/**
-	 * Specifies attach artifact Classifier, Ignored if attach=false
+	 * Specifies attach artifact Classifier, ignored if {@link #attach} is 'false'.
 	 *
 	 * @parameter default-value="small"
 	 */
 	private String attachArtifactClassifier;
 
 	/**
-	 * Set to false to exclude the attachArtifactClassifier from the Artifact final name. Default value is true.
```

Comment:
Note: Omitted the "Default value is true." because the generated site automatically describes this based on the `@parameter` tag.